sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -312,6 +312,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val STREAMING_META_DATA_NUM_RETRIES = buildConf("spark.sql.streaming.metadata.num.retries")
+    .doc("The number of meta data log retries.")
+    .intConf
+    .checkValue(_ > 0, "The minimum number of metadata retries " +
+      "must be a positive integer.")
+    .createWithDefault(3)
+
```

Comment:
Could you put this near the other `spark.sql.streaming.XXX` configuration in this file?